neurons/validator.py

```diff
@@ -335,12 +336,113 @@ def prepare_event_for_logging(event: Dict):
         if protein is not None and job.active is False:
             protein.remove_pdb_directory()
 
+    async def sync_loop(self):
+        bt.logging.info("Starting sync loop.")
+        while True:
+            self.sync()
+            seconds_per_block = 12
+            await asyncio.sleep(self.config.neuron.epoch_length * seconds_per_block)
```

Comment:
so I don't actually think we need to wait this long. There are internal waiting conditions within the `sync` method itself. So we could change this to 1s